pkg/tsdb/elasticsearch/response_parser.go

```diff
@@ -711,14 +712,17 @@ func getErrorFromElasticResponse(response *es.SearchResponse) string {
 	json := simplejson.NewFromAny(response.Error)
 	reason := json.Get("reason").MustString()
 	rootCauseReason := json.Get("root_cause").GetIndex(0).Get("reason").MustString()
+	causedByReason := json.Get("caused_by").Get("reason").MustString()
 
 	switch {
 	case rootCauseReason != "":
 		errorString = rootCauseReason
 	case reason != "":
 		errorString = reason
+	case causedByReason != "":
+		errorString = causedByReason
 	default:
-		errorString = "Unknown elasticsearch error response"
+		errorString = fmt.Sprintf("Unknown elasticsearch error response. Response = [%v]", json)
```

Comment:
could you please only do the `case causedByReason` switch, and not change the `default` case?

i understand the motivation (provide more debugging info in an unforeseen situation), but we are very careful with what we return intoto the browser, in case it contains something sensitive. and considering the `case causedByReason` fixes the current problem, i recommend only changing that, it will make the PR merged easier 😄 . thanks!

Reply:
Would it be an option to write it to the server logs?

When I had the issue, I checked the browser console on the network tab and server logs, but couldn't find any information about the error.
I also checked elasticsearch errors, but also didn't see any error (this could of course be my configuration).